api/src/pcapi/core/providers/models.py

```diff
@@ -63,6 +59,12 @@ def isAllocine(self) -> bool:
 
         return self.localClass == local_providers.AllocineStocks.__name__
 
+    @property
+    def isCinemaProvider(self) -> bool:
+        from pcapi import local_providers
+
+        return self.localClass == local_providers.CDSStocks.__name__
```

Comment:
(détail) On devrait pouvoir utiliser `CINEMA_PROVIDER_NAMES` : 

```suggestion
        return self.localClass in constants.CINEMA_PROVIDER_NAMES
```